bin/unleash.js

```diff
@@ -36,6 +44,12 @@ const argv = require('yargs')
         type: 'string',
     }).argv;
 
+if (argv.databaseUrlFile) {
+    // Where options are processed, the DATABASE_URL_FILE env variable takes precedence over databaseUrl.
+    process.env.DATABASE_URL_FILE = argv.databaseUrlFile;
```

Comment:
I don't think we should set environment variables. This can lead to unknown behaviour depending on how modules are loaded, cached etc.

Either pass it as an official argument as part of options or read the file and pass the content 
as `databaseUrl`.

Reply:
Done.

Reply:
thanks ;D